### DISCUSSION TITLE
Motion+ Made Yet?

### DISCUSSION THREAD

**Discussion by yuri-kiss** [TurboWarp/extensions#671] [Q&A] [answered]
Is anyone planning on making this or has any made it but has it unfinished?
(title looked misleading so I changed it)

**Comment by LilyMakesThings** [comment, marked as answer]
I started work on this with TheShovel yesterday. I didn't make it because I hadn't actually thought of any features that would make it worthwhile (and weren't from the other TurboWarp mod that shall not be named), but we actually came up with 2 really good ideas for it.

I'll let you know how that goes.

**Reply by yuri-kiss** [reply]
> I'll let you know how that goes.
Thanks!

Btw I am guessing you already have this but just in case: could you try to add a `point towards(x, y)` block?

**Reply by LilyMakesThings** [reply]
That was one of the first blocks added, don't worry :)

**Reply by yuri-kiss** [reply]
:D, Yay I will be waiting for it, I hope development goes smoothly 👍

**Comment by MA-master-sec** [comment]
I'm also making.   :  )
